user_sync/rules.py

```diff
@@ -154,15 +162,14 @@ def run(self, directory_groups, directory_connector, umapi_connectors):
             load_directory_stats.log_start(logger)
             self.read_desired_user_groups(directory_groups, directory_connector)
             load_directory_stats.log_end(logger)
-            should_sync_umapi_users = True
-        else:
-            # no directory users to sync with
-            should_sync_umapi_users = False
 
-        umapi_stats = JobStats("Sync Umapi", divider="-")
+        umapi_stats = JobStats('Sync Umapi' if self.sync_umapi else 'Push to Umapi', divider="-")
```

Comment:
This isn't a big issue, but I think we write UMAPI all caps in most places, including the documentation.  In addition, is that the term we use as the destination or is it more "Adobe" or "Adobe User Management".